### DISCUSSION TITLE
Adding new classes and images to already created task

### DISCUSSION THREAD

**Discussion by gnajaganathan** [cvat-ai/cvat#4063] [Ideas]
Assume that i want to annotate the fruits. Fruits lists are {apple,orange,pine apple, water melon, papaya, ...., cherry}. I have created a new task in cvt and uploaded my images. Now i want add new fruits classes {banana, mango, jack fruits ,...} to above created task. Is there any way to add new classes  to already created task?

**Comment by bsekachev** [comment]
Hi, 

You can easily add new classes (labels) to a task. You can't add new images to the task.

I would suggest in this case to create a project "Fruits", and move the first task there. Then add to this project new labels (banana, mango, ...) and create a new task with new images, annotate it as usual. In the future you can easily get annotations for all the tasks as a dataset. And you can always append new data to the project
